### PR TITLE
Table sort

### PR DESCRIPTION
**Pull request by ryanballa**
Adding the ability to sort a table by a column. This is not hooked up to the UI yet. This is more of a sanity check before I continue.

### CODE REVIEW

**Comment by ryanballa** [comment]
@KevinMarren happy to go over this in-person if you'd like.

**Comment by ryanballa** [comment]
#12

File: src/components/Table.jsx [hunk 1]
```diff
@@ -18,6 +18,14 @@ export default class Table extends Base {
     headers: React.PropTypes.object,
     multiSelectable: React.PropTypes.bool,
     noRecordsText: React.PropTypes.string,
+    orderBy: React.PropTypes.shape({
```
Comment (KevinMarren) [inline src/components/Table.jsx]: Just skimmed, curious why no spec in ./specs/components/Table.spec.jsx

Will look more tomorrow.

Comment (ryanballa) [inline src/components/Table.jsx]: I added a spec, still need to hookup clicking the header.

**Comment by ryanballa** [comment]
I've added tests in Table.spec with the latest commit @milemilo .

**Pull request merged**